appservice/src/tree/AppSettingTreeItem.ts

```diff
@@ -77,4 +77,28 @@ export class AppSettingTreeItem extends AzureTreeItem<ISiteTreeRoot> {
         this._hideValue = !this._hideValue;
         await this.refresh();
     }
+
+    public async toggleSlotSetting(): Promise<void> {
+        const slotSettings: SlotConfigNamesResource = await this.root.client.listSlotConfigurationNames();
+        if (slotSettings.appSettingNames) {
```

Comment:
I feel like I should still be able to toggle slot settings even if `slotSettings.appSettingNames` isn't defined